### PR TITLE
add file destination mode and rst format

### PR DESCRIPTION
**Pull request by JLazarte**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -127,3 +127,5 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+.vscode/
```
Comment (fernandezpablo85) [inline .gitignore]: this belongs in [global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c)

**Review by fernandezpablo85 — CHANGES_REQUESTED** [review]
These are awesome features! Thanks @JLazarte !

Can we please split this PR into two?

- .rst support
- output to a different path

and we can discuss changes on each?

File: .gitignore (hunk 1 above)
Comment (JLazarte) [reply .gitignore]: yes, I will remove it in the other pull request

**Comment by JLazarte** [comment]
I already separate every change in a new branch, the "rst support" change is already in a pull request in this repo, but for the others I will create a pull request in my fork (I dont see other way to make a chain of pull requests, every change need the previus one)

**Pull request closed without merging**